<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VisualArduino</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWebView" name="webView">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVerify"/>
   <addaction name="actionSend"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionVerify">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/check_alt_24x24.png</normaloff>:/icons/check_alt_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
   <property name="toolTip">
    <string>Verify</string>
   </property>
  </action>
  <action name="actionSend">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/arrow_right_alt1_24x24.png</normaloff>:/icons/arrow_right_alt1_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="toolTip">
    <string>Send</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionVerify</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionVerify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionVerify()</slot>
 </slots>
</ui>
